docs/network/transfers.md

```diff
@@ -135,23 +135,16 @@ Transactions are signed using the [EIP-712](https://github.com/ethereum/EIPs/blo
 }
 ```
 
-Note, the child chain server collects fees for sending a transaction. The fee can be paid in a variety of supported tokens by the network. To get more details on how the fees are defined, please refer to [Fees](https://docs.omg.network/fees).
+The child chain server collects fees for sending a transaction. The fee can be paid in a variety of supported tokens by the network, the fee amount is automatically calculated by the child chain. To get more details on how the fees are defined, please refer to [Fees](/network/fees). Note, testnet and mainnet might support different currencies for fees.
```

Comment:
the fee amount is only automatically calculated by the childchain when using the `childchain.createTransaction` method. if the user was to create the transaction themselves, they would have to fetch the fee amount and form the transaction accordingly.

Reply:
We do use `createTransaction` in this sample. I just haven't noticed that the Fee object doesn't have the amount property, which is why it was causing some errors.